### DISCUSSION TITLE
How to convert gettting started tutorial to be used with mysql?

### DISCUSSION THREAD

**Discussion by realtebo** [diesel-rs/diesel#3275] [Q&A]
One step at time

How can I convert this into a mysql version?

```
pub fn create_post<'a>(conn: &MysqlConnection, title: &'a str, body: &'a str) -> Post {
    use schema::posts;

    let new_post = NewPost {
        title: title,
        body: body,
    };

    diesel::insert_into(posts::table)
        .values(&new_post)
        .get_result(conn)
        .expect("Error saving new post")
}
```

**Comment by weiznich** [comment]
You can find the corresponding code for the getting started guide for each backend in the example directory in this repo. (Be sure to checkout the right branch for your diesel version). [Direct link for mysql + 1.4.x](https://github.com/diesel-rs/diesel/tree/v1.4.4/examples/mysql)
